mdbook/src/05-meet-your-software/debug-it.md

```diff
@@ -49,9 +49,19 @@ Remote debugging using :1337
 157     #[derive(Copy, Clone, Debug)]
 ```
 
-**NOTE**: The example in the repository for this chapter may
-change over time. Line numbers and other source details may
-thus be different from what is shown here and below.
+> **NOTE**: The example in the repository for this chapter may change over time. Line numbers and
+> other source details may thus be different from what is shown here and below.
+>
+> If the program fails to halt after starting, and you end up somewhere deeper in the program like
+> the following, then try running `monitor reset halt` to reset. This is due to [a bug](https://github.com/probe-rs/probe-rs/issues/3438)
+> in `probe-rs`, see [issue #27](https://github.com/rust-embedded/discovery-mb2/issues/27) for
+> more details.
+> ```shell
+> (gdb) target remote :1337
+> Remote debugging using :1337
+> init::__cortex_m_rt_main () at mdbook/src/05-meet-your-software/examples/init.rs:19
+> 19              asm::nop();
+> ```
```

Comment:
Not seeing the actual `monitor reset halt` in this terminal fragment?

Reply:
Added in!